modules/storage/src/main/scala/zio/gcp/storage/package.scala

```diff
@@ -25,121 +25,178 @@ package object storage {
 
   type Storage = Has[Storage.Service]
 
-  def batch: RIO[Storage, StorageBatch]                           = RIO.accessM(_.get.batch)
+  def batch: RIO[Storage, StorageBatch] = RIO.accessM(_.get.batch)
+
```

Comment:
had to add extra lines between methods as the latest version of scalafmt made for some ugly formatting with the equal operators all aligned up

Reply:
I think you can either disable alignment globally, or do it locally [like this](https://scalameta.org/scalafmt/docs/configuration.html#-format-off).

Reply:
probably best a new line is introduced anyway as at some point I will document these accessors